helm/nginx-ingress-controller-app/templates/controller-metrics-service.yaml

```diff
@@ -5,12 +5,13 @@ metadata:
   namespace: {{ .Release.Namespace }}
   annotations:
     giantswarm.io/monitoring: "true"
```

Comment:
```suggestion
  annotations:
```

Should this be removed?

Reply:
It's still needed for the current Prometheus.

Reply:
I am contemplating a PR to _prometheus-config-controller_ to change the existing setup to also look for a label. That would allow replacing it and would make things cleaner straight away. I'm not sure though how many managed apps already use that annotation and so how many changes does that imply. Any ideas on this @sslavic?

Reply:
OK, clear, let's add for a start, clean up later.